src/systems/filecoin_vm/interpreter/_index.md

```diff
@@ -50,6 +50,36 @@ The sequence of executions for a tipset is thus summarised:
 - [... subsequent blocks ...]
 - cron tick 
 
+# Message validity and failure
+Every syntactically valid message can be processed and produce a receipt. However, execution may
```

Comment:
worth linking to message/index.id after `syntactically valid` so readers can quickly move between syntactic and semantic validity